pom.xml

```diff
@@ -233,7 +233,7 @@
 
                 <plugin>
                     <artifactId>maven-javadoc-plugin</artifactId>
-                    <version>2.10.4</version>
+                    <version>3.0.0-M1</version>
```

Comment:
Is this a temporary workaround until there is a GA release?

Reply:
Yes we waited for about 2 months and we don't know when the plugins will release. I think it's safe to upgrade enforcer and javadoc to milestone version